### DISCUSSION TITLE
Media Stream from Tone js/web audio API??

### DISCUSSION THREAD

**Discussion by Walrus92** [foobar404/Wave.js#33] [Q&A] [answered]
Hello there.
I'm making a step sequencer with Tone JS for a school project. 
Now that I have reached all the audio side I need to get that mediaStream in order to use it for Wave.js fromStream. 
How could I do that? Cause using UserMedia does not word. 

Thx a lot!

**Comment by foobar404** [comment, marked as answer]
fromStream is meant to be used with microphone data. If you have audio coming from an audio synthesizer like tone js, it likely wont work, it is absolutely possible to create animations from that kind of audio, but Wave.js doesn't handle it out of the box. See if you can jerry rig something where you pipe the audio into a html audio element, then use fromElement.
